frontend/src/actions/suggestions.js

```diff
@@ -7,119 +7,173 @@ export const getStorySuggestions = (story) => {
       story.sick === sicknessStatus.RECOVERED &&
-      story.tested === testStatus.POSITIVE,
+      story.tested === testStatus.POSITIVE &&
+      story.currentLocation === countries.USA,
     link:
       "https://www.redcrossblood.org/donate-blood/dlp/plasma-donations-from-recovered-covid-19-patients.html#donorform",
   };
 
-  return mostRelevant(story, [testedRecovered]);
-};
-
-const informationForUSCitizens = (story) => {
-  const baseCondition = story.countryOfOrigin === "United States of America";
+  return mostRelevant([testedRecovered]);
+});
 
-  const citizenInMexico = {
-    predicate: baseCondition && story.currentLocation === "Mexico",
+const informationForUSCitizens = withStory((story) => {
+  const usCitizenInMexico = {
+    predicate:
+      story.countryOfOrigin === countries.USA &&
+      story.currentLocation === countries.MEX,
     link: "https://mx.usembassy.gov/u-s-citizen-services/covid-19-information/",
   };
-  const citizenAbroad = {
+
+  return mostRelevant([usCitizenInMexico]);
+});
+
+const informationForUSTravelers = withStory((story) => {
+  const usCitizenAbroad = {
     predicate:
-      baseCondition && story.currentLocation !== "United States of America",
+      story.countryOfOrigin === countries.USA &&
+      ![countries.USA, countries.MEX].includes(story.currentLocation),
     link:
       "https://travel.state.gov/content/travel/en/international-travel/emergencies/what-state-dept-can-cant-do-crisis.html",
   };
 
-  return mostRelevant(story, [citizenInMexico, citizenAbroad]);
-};
+  return mostRelevant([usCitizenAbroad]);
+});
 
-const mostRelevantOrElse = (story, options) => (defaultLink) => {
-  return mostRelevant(story, options) || defaultLink;
-};
+const preventGettingSick = withStory((story) => {
+  const notSick = {
+    predicate: story.sick === sicknessStatus.NOT_SICK,
+    link:
+      "https://www.cdc.gov/coronavirus/2019-ncov/prevent-getting-sick/index.html",
+  };
+  return mostRelevant([notSick]);
+});
+
+const whenSick = withStory((story) => {
+  const sick = {
+    predicate: story.sick === sicknessStatus.SICK,
+    link:
+      "https://www.cdc.gov/coronavirus/2019-ncov/if-you-are-sick/steps-when-sick.html",
+  };
+  return mostRelevant([sick]);
+});
+
+const moreInfoAboutCovid = withStory((story) => {
+  const moreAboutCovid = "https://www.cdc.gov/coronavirus/2019-nCoV/index.html";
+  const currentlyOnUS = {
+    predicate: story.currentLocation === countries.USA,
+    link: moreAboutCovid,
+  };
+  const usCitizen = {
+    predicate: story.countryOfOrigin === countries.USA,
+    link: moreAboutCovid,
+  };
+  return mostRelevant([currentlyOnUS, usCitizen]);
+});
 
-const mostRelevant = (story, options) => {
-  const option = story && options.find((op) => op.predicate);
+const mostRelevant = (options) => {
+  const option = options.find((op) => op.predicate);
   return option && option.link;
 };
+
+const countries = {
+  USA: "United States of America",
+  MEX: "Mexico",
```

Comment:
I'm hesitant about this, but as long as we don't loose sight that this will eventually be in the backend (hopefully) it should be ok